examples/backup_vm.py

```diff
@@ -279,13 +293,6 @@ def add_start_backup_args(parser):
         help="Disk UUID to backup. May be used multiple times to backup "
              "multiple disks. If not specified, backup all VM disks.")
 
-    parser.add_argument(
-        "--skip-download",
-        dest="download_backup",
-        action="store_false",
-        help="Download full backup disks should be skipped. If specified, VM backup will "
-             "start and stop without downloading the disks.")
-
```

Comment:
The change makes sense, but it will be better not to duplicate the option. I don't know if @shenitzky will have time to send another version, so I can post the next version.